articles/zero-trust-attestation-with-fleet

```diff
@@ -0,0 +1,68 @@
+
+### The policy engine
+Fleet allows security teams to write policies which reflect the company's access requirements. Under the hood, osquery regularly checks these policies and sends the results to the Fleet server. 
+
+For example, a basic policy at nearly all organizations is that the device hard disk is encrypted. In Fleet, this can be accomplished with the following policy:
+```
+SELECT 1 
+FROM disk_encryption 
+WHERE user_uuid IS NOT '' 
+  AND filevault_status = 'on'
+;
+```
+
+Once the policy results are saved in the Fleet server, each time a user attempts to access a protected resource, the system can check whether the device is in compliance and deny access where appropriate. Specifically, the relevant Fleet API endpoint is [`GET /api/v1/fleet/hosts/{id}`](https://fleetdm.com/docs/using-fleet/rest-api#get-host). This endpoint will respond with all of the policies for the supplied device ID and whether it passed or failed.
+
+### Self-remediation communication via Fleet Desktop
+If a user is blocked from access, what then? A complete zero trust must also provide information to users about why they were blocked and what they can do to address it. 
+
+When writing a policy, Fleet includes a field called "remediation." This is where security and IT teams should include plain-language end-user-facing steps to resolve the issue. 
+
+Fleet provides a desktop application called Fleet Desktop, which lives in the menubar. Fleet Desktop gives the user access to a Fleet webpage called "My Device," which, among others, features policy information. A user who has been denied access can visit this page to see the remediation instructions. This way, IT and security teams are partners of users in regaining access to company-owned resources. This self-remediation first approach reduces also the volume of help requests. 
+
+To learn more about installing Fleet Desktop, visit [here](https://fleetdm.com/docs/using-fleet/fleet-desktop).
+
+### Bring-your-own authentication provider
+One piece of the puzzle that you should bring, however, is your authentication provider. When your users request access, your authentication system should ping the Fleet API to check for policy compliance and make decisions about blocking based on the results. When presenting the access denied page, be sure to point users to Fleet Desktop's "My Device" page to help them find remediation instructions. 
+
+## Considerations for zero trust
+
+### Scalability and latency of the policy engine
+Since the request for policies will happen every time the user attempts to access company resources, this will generate a lot of requests. The Fleet policies endpoint has been optimized for quick pings. However, depending on the size of your organization, this will require tuning and upscaling the servers   to meet your organization's situation. 
```

Comment:
```suggestion
Since the request for policies will happen every time the user attempts to access company resources, this will generate a lot of requests. The Fleet policies endpoint has been optimized for high request volume. However, depending on the size of your organization, this may require tuning and upscaling the servers.
```